src/Integration.UnitTests/Telemetry/TelemetryManagerTests.cs

```diff
@@ -676,6 +679,28 @@ public void TaintIssueInvestigatedRemotely_CounterIncremented(int previousCounte
             telemetryRepositoryMock.Verify(x => x.Save(), Times.Once);
         }
 
+        [TestMethod]
+        [DataRow(true)]
+        [DataRow(false)]
+        public void SendPayload_SendsVsVersion(bool isVersionNull)
```

Comment:
Supplementary test to `TelemetryHelper_CreatePayload.cs`. This test just checks that the version from IVsVersionProvider is passed through to TelemetryHelper, but unfortunately TelemetryHelper is a static class, so I can't easily Verify that it was called with the given argument.